sonic-net/sonic-mgmt
# [m0] Refactor announce routes for m0

### Description of PR


Summary:
Fixes # (issue)

### Type of change



- [x] Testbed and Framework(improvement)


### Back port request

### Approach
#### What is the motivation for this PR?
In previous, announce routes for m0 refers to the existing code, some of which function and variable names may cause misunderstanding.

#### How did you do it?
1. Rename m0 template file.
2. Refactor announce routes for m0.

#### How did you verify/test it?
1. Run add-topo and deploy-mg to deploy m0 topo.
2. Run cmd 'show ip bgp sum' and 'show ip bgp nei 10.0.0.57 received-routes' to verify routes are announced correctly.

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
